src/OpenTelemetry/Metrics/MetricPoint.cs

```diff
@@ -21,37 +21,45 @@
 
 namespace OpenTelemetry.Metrics
 {
+    /// <summary>
+    /// Stores details about a metric data point.
+    /// </summary>
     public struct MetricPoint
     {
         private readonly AggregationType aggType;
+
         private readonly HistogramBuckets histogramBuckets;
 
         // Represents either "value" for double/long metric types or "count" when histogram
-        private MetricPointValueStorage primaryValue;
+        private MetricPointValueStorage runningValue;
+
+        // Represents either "value" for double/long metric types or "count" when histogram
+        private MetricPointValueStorage snapshotValue;
 
-        // Represents either "lastValue" for double/long metric types when delta or "sum" when histogram
```

Comment:
We could still have this comment for this field.

Reply:
I can add it, but I don't think it is really needed. Because the field name on `HistogramBuckets` is `RunningSum`/`SnapshotSum` which more clearly conveys the (single) intention/purpose. The reason I have the comments on `MetricPoint` is the names `runningValue`/`snapshotValue` don't really indicate they also stand in for the "Count" storage. Those fields are multi-purpose.